scripts/index-demo-page.js

```diff
@@ -1,6 +1,34 @@
 /* global window document */
-/* eslint-disable no-var, object-shorthand, no-console */
+/* eslint-disable vars-on-top, no-var, object-shorthand, no-console */
 (function(window) {
+
+  const hlsOptGroup = document.querySelector('[label="hls"]');
+  const dashOptGroup = document.querySelector('[label="dash"]');
+  const liveOptGroup = document.querySelector('[label="live"]');
+  const llliveOptGroup = document.querySelector('[label="low latency live"]');
+
+  // get the sources list squared away
+  window.fetch('./scripts/sources.json')
```

Comment:
Will we lose IE11 support for the test page with this?

Reply:
Maybe, probably.